py/src/ai/h2o/sparkling/H2OContext.py

```diff
@@ -290,7 +244,7 @@ def as_spark_frame(self, h2o_frame, copy_metadata=True):
             df._h2o_frame = h2o_frame
             return df
 
-    def as_h2o_frame(self, dataframe, framename=None, full_cols=100):
+    def as_h2o_frame(self, dataframe, framename=None, full_cols=-1):
```

Comment:
I've made this change for two reasons:

- In rest mode, we don't have a java H2OFrame and we can't call `.numCols()` https://github.com/h2oai/sparkling-water/pull/1647/files#diff-9143aa3a0fb6ae5a60db472c5edacf9dL68
- -1 is default value for other methods of python client. http://docs.h2o.ai/h2o/latest-stable/h2o-py/docs/frame.html#h2o.frame.H2OFrame.get_frame I've thought it would be nice if we were consistent accross Python APIs.

If you think it's crucial to have default value `100`, i could try to progate number of columns from REST objects.

Reply:
I like to be consistent with the H2O. I think we have limited this to 100 in Sparkling Water case that loading really big frames with a lot of columns was really slow. But I think we are now using light endpoint which should be fine